content/features/featuresDeepDive/webXR/WebXRMovement.md

```diff
@@ -0,0 +1,162 @@
+---
+title: WebXR Movement
+image:
+description: Learn about how to move in a WebXR scene
+keywords: babylon.js, diving deeper, WebXR, VR, AR, movement, locomotion
+further-reading:
+video-overview:
+video-content:
+---
+
+## Movement Module
+
+The movement feature allows movement in the scene with the controllers. It is separate from the teleportation module and designed to not be used at the same time.
+
+In order to enable this feature you must first ensure that teleportation is not switched on.
+
+Teleportation can be disabled in the default XR experience:
+
+```javascript
+const xr = await scene.createDefaultXRExperienceAsync({
+  disableTeleportation: true,
+});
+```
+
+You can also disable teleportation manually before enabling the movement feature.
+
+```javascript
+const featureManager = xrHelper.baseExperience.featuresManager;
+featureManager.disableFeature(BABYLON.WebXRFeatureName.TELEPORTATION);
+```
+
+The movement feature enables the camera's position to be changed with the controllers.
+Default configuration (can be changed in options below):
+
+1. Left Controller - Rotation counter + clockwise
+2. Right Controller - Move forward/backwards/left/right
+
+```javascript
+const featureManager = xrHelper.baseExperience.featuresManager;
+
+featureManager.enableFeature(BABYLON.WebXRFeatureName.MOVEMENT, "latest", {
+  xrInput: xrHelper.input,
+});
+```
+
+<Playground id="#AZML8U" title="Movement with controllers" description="A simple example of controller movement with collisions and gravity" image="/img/how_to/xr/xr-movement-playground.png"/>
+
+### Movement Configuration Options
+
+The current options for the plugin can always be found at the [WebXR movement feature source code](https://github.com/BabylonJS/Babylon.js/tree/master/packages/dev/core/src/XR/features/WebXRControllerMovement.ts#L20). Most of them will be explained here.
+
+One important option is `movementOrientationFollowsViewerPose`, which defaults to `true`. When configured to `false` the forward direction will not adjust with the viewer pose, but only to the controller rotation. This can be useful for some experiences where the viewer pose should not affect movement direction. ie: If you are moving forward and look left that you should not drift left, but continue straight. You can try out in the above playground by changing the option.
+
+The controllers themselves can be configured differently. If you want to swap which hand controls rotation vs. movement, for example, then override the option `customRegistrationConfigurations` array. The movement state rotation and movement is handled automatically by the feature and ignored accordingly if rotation (`rotationEnabled`) or movement (`movementEnabled`) are not enabled and will automatically be adjusted to speed (`rotationSpeed` & `movementSpeed`).
```

Comment:
"vs. movement, for example," -> "vs. movement for example,"

Reply:
```suggestion
The controllers themselves can be configured differently. If you want to swap which hand controls rotation vs. movement for example, then override the option `customRegistrationConfigurations` array. The movement state rotation and movement is handled automatically by the feature and ignored accordingly if rotation (`rotationEnabled`) or movement (`movementEnabled`) are not enabled and will automatically be adjusted to speed (`rotationSpeed` & `movementSpeed`).
```